e2e/README.md

```diff
@@ -1,22 +1,21 @@
 These e2e tests depend on emulating or attaching a device.
 
-1) setup android stack
-you can do this with android studio. create a api-level 30 x86_64 device and name it 30_ga_64
-alternatively, if you dont have android studio, this can help: https://github.com/fluidvanadium/androidstacker
+0) install prerequisites
+# 1) docker
+# 2) sdkmanager (android commandline tools: https://dl.google.com/android/repository/commandlinetools-linux-9477386_latest.zip)
 
-2) install the necessary emulator with scripts/install_emulator_for_e2e.sh
+1) set up emulator
 
-3) build zingo-mobile
-`$ sh scripts/setup.sh`
+`./scripts/start_flow_emulator.sh -a x86_64`
 
-4) run e2e tests
-`$ sh scripts/run_e2e.sh`
+2) 
 
-or to run with specifications:
-check installed emulators with 
-`$ emulator -list-avds`
-compare to the configuration aliases in `.detoxrs`
 pick a test called `e2e/TESTNAME.test.js`
-`yarn detox build TESTNAME -c CONFIGURATION`
-`yarn detox test TESTNAME -c CONFIGURATION`
+`yarn detox build -c android.emu.debug`
+`yarn detox test TESTNAME -c android.emu.debug`
+
+alt)
+`./scripts/start_flow_emulator.sh -a armeabi-v7a`
+`yarn detox test -c android.emu.32`
```

Comment:
test should be after build right?

also i think this should be the default and you should explain how to run a specific test afterwards, what do you think?

Reply:
oops, i guess i wasnt clear

Reply:
test is still before build...

also what do you think about having the instructions explain how to run all tests and then have a:
`run a specific test)`
pointer at the end ?